### DISCUSSION TITLE
Branch prefix per dependency (group)

### DISCUSSION THREAD

**Discussion by biergit** [renovatebot/renovate#9631] [Request Help] [answered]
Is this possible? Same question about commit message prefix? We require certain branch and commit prefixes to connect to Jira issues.
In some cases its not ideal to only use one prefix per repository e.g when you have a repository that mostly integrates dependencies from different teams which use different Jira boards and thus need their own issue to integrate their versions into the integration repository.

**Comment by rarkins** [comment, marked as answer]
Not possible. The branch prefix is necessary for efficient git fetching.
